docs/openapi.yaml

```diff
@@ -488,6 +488,46 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+
+  /import-candidate/{identifier}:
+    get:
+      summary: Fetch Import candidate by Identifier
+      tags:
+        - external
```

Comment:
You dont need this tag unless you want the call to be documented for externals. 
I also see you  dont have the "secuirty" property. Do you want this to be publicly accessible?